libflux/flux-core/Cargo.toml

```diff
@@ -41,7 +41,7 @@ flatbuffers = "2.1.1"
 fnv = "1.0.7"
 libflate = "1.2.0"
 log = "0.4.16"
-lsp-types = { version = "0.93", optional = true }
+lsp-types = { version = "0.92", optional = true }
```

Comment:
We only use `Position` in `flux-core` which is very stable across `lsp-types` version so we could specify a version range here to maybe avoid some churn.